submission-scripts/script-templates/Makefile

```diff
@@ -605,6 +607,27 @@ define create-spark-with-tachyon-and-hdfs
 	sed -i -e "s/TACHYON_SETUP=.*/TACHYON_SETUP=yes/" $(MAGPIE_SPARK_WITH_TACHYON_AND_HDFS)
 endef
 
+define create-spark-with-zeppelin
+	echo "Creating magpie.$(SCHED)-spark-with-zeppelin"
+	cat 	magpie-header \
+		magpie-config-$(SCHED) \
+		magpie-magpie-customizations-substitution \
+		magpie-general-configuration \
+		magpie-general-configuration-python \
+		magpie-spark \
+		magpie-zeppelin \
+		magpie-run-job-header-substitution \
+		magpie-run-job-$(DIST) > $(MAGPIE_SPARK_WITH_ZEPPELIN)
+	$(call common-substitution, ${MAGPIE_SPARK_WITH_ZEPPELIN})
+	$(call common-additions, ${MAGPIE_SPARK_WITH_ZEPPELIN}, zeppelin)
+	sed -i -e "s/MAGPIE_JOB_TYPE=\"\(.*\)\"/MAGPIE_JOB_TYPE=\"spark\"/" $(MAGPIE_SPARK_WITH_ZEPPELIN)
+	sed -i -e "s/SPARK_SETUP=.*/SPARK_SETUP=yes/" $(MAGPIE_SPARK_WITH_ZEPPELIN)
+	sed -i -e "s/^# export SPARK_LOCAL_SCRATCH_DIR=/export SPARK_LOCAL_SCRATCH_DIR=/g" $(MAGPIE_SPARK_WITH_ZEPPELIN)
+	sed -i -e 's/export SPARK_MODE="\(.*\)"/export SPARK_MODE="setuponly"/' $(MAGPIE_SPARK_WITH_ZEPPELIN)
```

Comment:
So my understanding is that if Zeppelin is configured correctly, it will just use the Spark that is already running.

https://zeppelin.apache.org/docs/0.6.0/interpreter/spark.html#1-export-spark_home

Are we not doing something correctly?